packages/e2e-tests/src/tests/WrappedKeys.spec.ts

```diff
@@ -0,0 +1,1211 @@
+    ); // 10 mins expiry
+
+    const { decryptedPrivateKey } = await exportPrivateKey({
+      pkpSessionSigs: pkpSessionSigsExport!,
+      litNodeClient: devEnv.litNodeClient!,
+      network: 'solana',
+      id,
+    });
+
+    expect(decryptedPrivateKey).toEqual(privateKey);
+  });
+
+  it('Eth Sign Tx', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    const pkpSessionSigs = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    const privateKey = ethers.Wallet.createRandom().privateKey;
+
+    const { pkpAddress, id } = await importPrivateKey({
+      pkpSessionSigs: pkpSessionSigs!,
+      privateKey,
+      litNodeClient: devEnv.litNodeClient!,
+      publicKey: '0xdeadbeef',
+      keyType: 'K256',
+      memo: 'Test Key',
+    });
+
+    const alicePkpAddress = alice.authMethodOwnedPkp?.ethAddress;
+
+    expect(pkpAddress).toEqual(alicePkpAddress);
+
+    const pkpSessionSigsSigning = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    const unsignedTransaction = getBaseTransactionForNetwork({
+      network: devEnv.litNodeClient?.config.litNetwork!,
+      toAddress: alice.wallet.address,
+    });
+
+    const signedTx = await signTransactionWithEncryptedKey({
+      pkpSessionSigs: pkpSessionSigsSigning!,
+      network: 'evm',
+      unsignedTransaction,
+      broadcast: false,
+      litNodeClient: devEnv.litNodeClient!,
+      id,
+    });
+
+    expect(!ethers.utils.isHexString(signedTx)).toBeTruthy();
+  });
+
+  it('Eth Sign Message', async () => {
+    const alice = await devEnv.createRandomPerson();
+
+    const pkpSessionSigs = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    const privateKey = ethers.Wallet.createRandom().privateKey;
+
+    const { pkpAddress, id } = await importPrivateKey({
+      pkpSessionSigs: pkpSessionSigs!,
+      privateKey,
+      litNodeClient: devEnv.litNodeClient!,
+      publicKey: '0xdeadbeef',
+      keyType: 'K256',
+      memo: 'Test Key',
+    });
+
+    const alicePkpAddress = alice.authMethodOwnedPkp?.ethAddress;
+    expect(pkpAddress).toEqual(alicePkpAddress);
+
+    const pkpSessionSigsSigning = await getPkpSessionSigs(
+      devEnv,
+      alice,
+      undefined,
+      new Date(Date.now() + 1000 * 60 * 10).toISOString()
+    ); // 10 mins expiry
+
+    const unsignedStringMessage = 'This is a test message';
+
+    const signature = await signMessageWithEncryptedKey({
+      pkpSessionSigs: pkpSessionSigsSigning!,
+      network: 'evm',
+      messageToSign: unsignedStringMessage,
+      litNodeClient: devEnv.litNodeClient!,
+      id,
+    });
+
+    console.log('signature');
+    console.log(signature);
```

Comment:
:hocho: we removed this from upstream because it was so spammy (and `expect()` will print the value if it doesn't match what we expect!)